marshal.go

```diff
@@ -1541,21 +1547,27 @@ func unmarshalMap(info TypeInfo, data []byte, value interface{}) error {
 	if len(data) < 2 {
```

Comment:
Do you still need this < 2 check? Might get away without it :-)

Reply:
I missed that one, thanks :) Removed now.